packages/perseus/src/widgets/__tests__/numeric-input_test.jsx

```diff
@@ -26,7 +26,10 @@ import {
 import type {APIOptions} from "../../types.js";
 import type {Rubric} from "../numeric-input.jsx";
 
-const renderQuestion = (question, apiOptions: APIOptions) => {
+const renderQuestion = (
+    question,
+    apiOptions: APIOptions = Object.freeze({}),
```

Comment:
That's interesting, `numeric-input_test` isn't using `renderQuestion` from `renderQuestion.jsx` like `radio_test.jsx`.

- If it's a relatively small change, maybe this would be a good opportunity to change this
- It it's not, should we make a ticket to circle back?